src/server/worldserver/worldserver.conf.dist

```diff
@@ -2936,6 +2936,13 @@ Arena.AutoDistributePoints = 0
 
 Arena.AutoDistributeInterval = 7
 
+#
+#    Arena.GamesRequired
+#        Description: Number of arena matches team must participate in to be eligible for arena point distribution.
```

Comment:
```suggestion
#        Description: Number of arena matches teams must participate in to be eligible for arena point distribution.
```
Number or amount? i don't remember

Reply:
both wordings are used about equally in the .conf